tests/test_all_fields.py

```diff
@@ -93,8 +93,9 @@ def test_run(self):
                 if stream == 'abandoned_checkouts':
                     expected_all_keys.remove('billing_address')
                 elif stream == 'orders':
-                    # No field named 'order_adjustments' present in the 'order' object
+                    # No field named 'order_adjustments', 'total_price_usd' present in the 'order' object
                     #   Documentation: https://shopify.dev/api/admin-rest/2021-10/resources/order#resource_object
-                    expected_all_keys.remove('order_adjustments')
+                    known_missing_fields = ['order_adjustments','total_price_usd']
```

Comment:
Since `known_missing_fields` is just getting converted into a set on the next line, we can just create it as a set and not do the conversion